samples/kafka_confluent/sender/main.go

```diff
@@ -26,9 +28,47 @@ func main() {
 	sender, err := confluent.New(confluent.WithConfigMap(&kafka.ConfigMap{
 		"bootstrap.servers": "127.0.0.1:9092",
 	}), confluent.WithSenderTopic(topic))
-
+	if err != nil {
+		log.Fatalf("failed to create protocol, %v", err)
+	}
 	defer sender.Close(ctx)
 
+	// Listen to all the events on the default events channel
+	// It's important to read these events otherwise the events channel will eventually fill up
+	go func() {
```

Comment:
nit: you're not waiting for this goroutine to finish before returning from `main`- which could prevent your goroutine from printing anything. typically use `https://pkg.go.dev/golang.org/x/sync/errgroup`

Reply:
Done